crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -338,6 +338,7 @@ en:
         unknown: '%{count} Unknown'
         unready: '%{count} Not Ready'
         pending: '%{count} Pending'
+        crowbar_upgrade: '%{count} Upgrade'
```

Comment:
Currently not really important, But this is currently unused. I guess @nkrinner simply forgot to git-add the related changes to nodes_helper.rb to the PR.